docs/content/en/docs/getting-started/nutanix/nutanix-spec.md

```diff
@@ -251,6 +273,36 @@ __Example__:</br>
     <certificate string>
     -----END CERTIFICATE-----
 ```
+### failureDomains (optional)
+The list of failure domains used for the control plane nodes. Three failure domains are recommended. 
+>**_NOTE:_** All the failure domain subnets must be on the same L2 network.
+
+### failureDomains[0].name
+The name of the failure domain.
+
+### failureDomains[0].cluster (required)
+Reference to the failure domain Prism Element cluster.
+
+### failureDomains[0].cluster.type (required)
+Type to identify the failure domain Prism Element cluster. (Permitted values: `name` or `uuid`)
+
+### failureDomains[0].cluster.name (`name` or `UUID` required)
```

Comment:
Do you mean: "failureDomains[0].cluster.name or failureDomains[0].cluster.uuid required"? Someone might think you mean this value could be set to `name` or `UUID`. It looks like this could only be set to `name`. Same question with other objects below that say: "(`name` or `UUID` required)". There are eight such objects here.

Reply:
Thanks for the feedback @chrisnegus. 
For the cluster object in a failure domain the name or the uuid are required. What would your guidance be to make this more clear? For example should we use the full path: "`failureDomains[0].cluster.name` or `failureDomains[0].cluster.UUID` required"?

Reply:
Yeah, maybe. My first impression was that `failureDomains[0].cluster.type` could be set to a name or a UUID, which I don't believe is the case here. So being explicit might be good.

Reply:
I have updated the documentation to use the full paths of the spec attributes. Please take a look.